src/generic/Matrix.jl

```diff
@@ -232,6 +232,11 @@ zero(x::MatrixElem, R::Ring, r::Int, c::Int) = zero!(similar(x, R, r, c))
 zero(x::MatrixElem, R::Ring=base_ring(x)) = zero(x, R, nrows(x), ncols(x))
 zero(x::MatrixElem, r::Int, c::Int) = zero(x, base_ring(x), r, c)
 
+@doc Markdown.doc"""
+    zero(a::Generic.MatrixElem)
+
+Construct the zero matrix in the same matrix space as `a`.
+"""
 function zero!(x::MatrixElem)
```

Comment:
This docstring seems to be added to the wrong method?

Reply:
Yes, you're totally right.